docusaurus.config.js

```diff
@@ -195,6 +195,12 @@ const config = {
     './src/plugins/segment',
     './src/plugins/launchdarkly',
     './src/plugins/sentry',
+    [
+      '@docusaurus/plugin-google-tag-manager',
```

Comment:
We use [plugin-google-gtag](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-gtag) plugin for [web3auth docs](https://github.com/Web3Auth/web3auth-docs/blob/master/docusaurus.config.ts#L216).